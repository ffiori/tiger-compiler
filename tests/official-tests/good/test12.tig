/* valid for and let */

let
	var a := 0
in 
	for i :=0 to 100 do (a :=a+1; printInt(a) ;()); 0
end
/* print 1...101 */
