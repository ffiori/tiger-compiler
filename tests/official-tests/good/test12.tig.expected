INT: 1
INT: 2
INT: 3
INT: 4
INT: 5
INT: 6
INT: 7
INT: 8
INT: 9
INT: 10
INT: 11
INT: 12
INT: 13
INT: 14
INT: 15
INT: 16
INT: 17
INT: 18
INT: 19
INT: 20
INT: 21
INT: 22
INT: 23
INT: 24
INT: 25
INT: 26
INT: 27
INT: 28
INT: 29
INT: 30
INT: 31
INT: 32
INT: 33
INT: 34
INT: 35
INT: 36
INT: 37
INT: 38
INT: 39
INT: 40
INT: 41
INT: 42
INT: 43
INT: 44
INT: 45
INT: 46
INT: 47
INT: 48
INT: 49
INT: 50
INT: 51
INT: 52
INT: 53
INT: 54
INT: 55
INT: 56
INT: 57
INT: 58
INT: 59
INT: 60
INT: 61
INT: 62
INT: 63
INT: 64
INT: 65
INT: 66
INT: 67
INT: 68
INT: 69
INT: 70
INT: 71
INT: 72
INT: 73
INT: 74
INT: 75
INT: 76
INT: 77
INT: 78
INT: 79
INT: 80
INT: 81
INT: 82
INT: 83
INT: 84
INT: 85
INT: 86
INT: 87
INT: 88
INT: 89
INT: 90
INT: 91
INT: 92
INT: 93
INT: 94
INT: 95
INT: 96
INT: 97
INT: 98
INT: 99
INT: 100
INT: 101
